ocs_ci/utility/powernodes.py

```diff
@@ -0,0 +1,132 @@
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.ocs.exceptions import UnexpectedBehaviour
+from ocs_ci.ocs.node import wait_for_nodes_status, get_worker_nodes, get_master_nodes
+from ocs_ci.ocs.ocp import wait_for_cluster_connectivity
+from ocs_ci.utility.utils import TimeoutSampler, exec_cmd
+from ocs_ci.utility.utils import get_ocp_version
+
+logger = logging.getLogger(__name__)
+
+
+class POWERNODES(object):
+    """
+    Wrapper for PowerNodes
+    """
+
+    isKVM = False
+
+    def __init__(self):
+        """
+        Class Initialization.
+        Run lsmod command to prepare isKVM variable.
+        """
+        cmd = "sudo /usr/sbin/lsmod"
+        result = exec_cmd(cmd)
+        if b"kvm" in result.stdout.lower():
+            self.isKVM = True
+        else:
+            self.isKVM = False
+        logger.info(f"iskvm check: {self.isKVM}")
+
+    def iskvm(self):
+        """
+        Verify if this environment is libvirt or kvm.
+        Args: None
+        Returns:
+            bool: True if this is kvm environment, False otherwise
```

Comment:
```suggestion
        Verify if this environment is libvirt or kvm.
        
        Returns:
            bool: True if this is kvm environment, False otherwise

```

Reply:
Done.